### DISCUSSION TITLE
Перевести теги/ключи/команды(хз как называется) из goodbyedpi-0.2.3rc3-2 в ByeDPI 1.2.0

### DISCUSSION THREAD

**Discussion by ForeverSleeping** [dovecoteescapee/ByeDPIAndroid#213] [Q&A]
Всем привет. В goodbyedpi используется строчка
`start "" goodbyedpi.exe start "" goodbyedpi.exe -5 -e 1 --reverse-frag --wrong-chksum --fake-from-hex 1603030135010001310303424143facf5c983ac8ff20b819cfd634cbf5143c0005b2b8b142a6cd335012c220008969b6b387683dedb4114d466ca90be3212b2bde0c4f56261a9801 --fake-gen 5 --set-ttl 3 -q --blacklist ..\russia-blacklist.txt --blacklist ..\russia-youtube.txt`

Может ли кто-нибудь её перевести для ByeDPI. Видел тут комментарий что кто-то из гудбая перевел в байдпиай и у него заработало. Сам я с инструкцией и той программы и этой не справился. Не говоря о том, что я даже не знаю, что такое "-5" - этого в документации не было.
Ростелеком

**Comment by saintmedium** [comment]
Запустил GoodCheck на компе, показал такие резщультаты

**Strategies with 27/27 successes:
[-q -e 1 --reverse-frag --set-ttl 8 --fake-from-hex 160303]
[-q -e 2 --reverse-frag --set-ttl 8 --fake-from-hex 160303]
[-q -e 2 --reverse-frag --wrong-chksum --set-ttl 8 --fake-from-hex 160303]**

как их перевести на ByeDPI?
